### ISSUE TITLE
Term Request: Mulberry-like corneal subepithelial deposits

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11743]
**Preferred term label:**
Mulberry-like corneal subepithelial deposits

**Synonyms**
* Mulberry-type corneal amyloid deposits
* Mulberry corneal nodules
* Mulberry subepithelial opacities

**Definition (free text, please give PubMed ID)**
Groups of small, multiple, elevated subepithelial gelatinous nodules arranged in a mulberry-like configuration on the anterior corneal surface. Histopathologically, they consist of mass subepithelial and anterior stromal amyloid deposits that stain with Congo red and antibodies to lactoferrin (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Subepithelial corneal opacities (HP:0008039)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Gelatinous drop-like corneal dystrophy (GDLD) (ORPHA:98957, OMIM:204870)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
